src/components/common/dropdown/DropdownTrigger.tsx

```
@@ -0,0 +1,23 @@
+import { useDropdown } from './DropdownContext';
+
+interface DropdownTriggerProps {
+  children?: ReactNode;
+  placeholder?: string;
+}
+
+const DropdownTrigger = ({ children, placeholder }: DropdownTriggerProps) => {
+  const { open, onOpenChange, value } = useDropdown();
+
+  return (
+    <button onClick={() => onOpenChange(!open)}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)